src/main/scala/com/itszuvalex/femtocraft/power/render/NodeCrystalRenderer.scala

```diff
@@ -34,7 +33,7 @@ trait NodeCrystalRenderer extends TileEntitySpecialRenderer {
     GL11.glColor4ub(color.red, color.green, color.blue, 220.toByte)
 
     val f2: Float = node.getWorldObj.getTotalWorldTime.toFloat + partialTime
-    (1 to 10).map(num => ("Gengon0" + (if (num < 10) "0") + num, num)).foreach { name =>
+    (1 to 10).map(num => ("Gengon0" + (if (num < 10) "0" else "") + num, num)).foreach { name =>
```

Comment:
This else statement is required now, as the new renderer handles strings a little differently, parsing this part as "()" if it doesn't return anything. Of course that leads to errors.